manage.py

```diff
@@ -50,6 +50,17 @@ def execute_sql_folder(path, processes):
             execute_sql_file(path)
 
 @manager.command
+def load_nicknames():
+    import pandas as pd
+    import sqlalchemy as sa
+    try:
+        table = sa.Table('ofec_nicknames', db.metadata, autoload_with=db.engine)
+        db.engine.execute(table.delete())
+    except sa.exc.NoSuchTableError:
+        pass
+    load_table(pd.read_csv('data/nicknames.csv'), 'ofec_nicknames')
+
+@manager.command
 def load_pacronyms():
```

Comment:
I notice that if I run update_all more than once I get an error-

```
sqlalchemy.exc.InternalError: (psycopg2.InternalError) cannot drop table ofec_pacronyms because other objects depend on it
DETAIL:  materialized view ofec_committee_fulltext_mv depends on table ofec_pacronyms
HINT:  Use DROP ... CASCADE to drop the dependent objects too.
 [SQL: '\nDROP TABLE ofec_pacronyms'] 
```